reposerver/repository/repository.go

```diff
@@ -1574,25 +1574,25 @@ func checkoutRevision(gitClient git.Client, revision string) error {
 		return status.Errorf(codes.Internal, "Failed to initialize git repo: %v", err)
 	}
 
-	// Some git providers don't support fetching commit sha
-	if revision != "" && !git.IsCommitSHA(revision) && !git.IsTruncatedCommitSHA(revision) {
-		err = gitClient.Fetch(revision)
-		if err != nil {
-			return status.Errorf(codes.Internal, "Failed to fetch %s: %v", revision, err)
-		}
-		err = gitClient.Checkout("FETCH_HEAD")
-		if err != nil {
-			return status.Errorf(codes.Internal, "Failed to checkout FETCH_HEAD: %v", err)
-		}
-	} else {
+	err = gitClient.Fetch(revision)
+
+	if err != nil {
+		log.Warnf("Failed to fetch revision %s: %v", revision, err)
```

Comment:
Please change it to `log.Info`. We are trying to use 'warning' logs only for something that the operator should act on.

Reply:
Done